cfgov/v1/models/filterable_list_mixins.py

```diff
@@ -52,29 +48,24 @@ def filterable_pages(self):
         if not site:
             return self.get_model_class().objects.none()
 
-        pages = self.get_model_class().objects.in_site(site).live()
-
-        if self.filterable_categories:
-            category_names = get_category_children(self.filterable_categories)
-            pages = pages.filter(categories__name__in=category_names)
-
+        queryset = self.get_model_class().objects.in_site(site).live()
         if self.filterable_children_only:
-            pages = pages.child_of(self)
+            queryset = queryset.child_of(self)
 
-        return pages
+        return self.get_model_class().objects.in_site(site).live()
```

Comment:
Is this a mistaken duplication from above? Eliminating the `.child_of(self)` means that, for example, [the Events page](http://localhost:8000/about-us/events/) pulls in every filterable page on the whole site.

```suggestion
        return queryset
```

Reply:
Indeed!